lib/tx/api.ts

```diff
@@ -69,6 +69,10 @@ function transactionExists(sdk) {
 }
 
 function postToTransaction(sdk, url, args, options?) {
+  options = { 
```

Comment:
Why don't we just put the change here (instead of the other places):
```
options = Object.assign({ withCredentials: true}, options)
```
Then the caller can pass { withCredentials: false } to disable